Quake/gl_model.h

```diff
@@ -362,7 +364,11 @@ typedef struct
 	char name[32];
 	bonepose_t inverse;
 } boneinfo_t;
-
+typedef struct
+{
+	float	xyz[4]; // only uses 3 but must be padded to 16 bytes
+	float	normal[4]; // ditto
```

Comment:
If the source normals are represented as 2 8-bit numbers, having 4 floats stored for each vertex seems overkill. We could halve the size of the vertex struct with 3 floats for the position and 4 bytes (3 + padding) for the normal.